### PR TITLE
Backport of PBC-1528: Fixed jenkins setup files - scheduler jobs weren't set up automatically

### PR DESCRIPTION
**Pull request by bohdan-turchyk-spryker**

### CODE REVIEW

File: config/Zed/cronjobs/jenkins.php [hunk 1]
```diff
@@ -1,9 +1,22 @@
 <?php
+
+/*
+ * Snippet to add on commands that we wnat to send logs to Cloudwatch.
+ * Issue and solution reported on https://spryker.atlassian.net/browse/PBC-1349.
+ */
+$sendOutputToJenkinsCommandSnippet = ' 2>&1 | tr "\n" "\0" | xargs -0 printf "${JOB_NAME} ${BUILD_NUMBER} %s\n" | tee -a /proc/1/fd/1';
```
Comment (gechetspr) [inline config/Zed/cronjobs/jenkins.php:7]: this is just an example? I can't find usage of it

Comment (bohdan-turchyk-spryker) [reply config/Zed/cronjobs/jenkins.php:7]: It should be appended to commands we want to log extract logs from.
Usually, we don't do that for all commands so it's not added to test command.

Comment (matweew) [reply config/Zed/cronjobs/jenkins.php:7]: but we already found that this approach does not work correctly in the local environment. + @k4emic introduced another approach with `src/bin/loggable.sh` wrapper which works better.

```
$loggable($console('queue:worker:start')),
```

so I suggest closing this PR.

Comment (bohdan-turchyk-spryker) [reply config/Zed/cronjobs/jenkins.php:7]: It also includes changes to install files as with current setup scheduler wasn't configured automatically due to wrong section name (true at the moment of development, not sure about that now)

**Pull request closed without merging**